test/stdio/async.js

```diff
@@ -65,6 +65,7 @@ const testMaxListeners = async (t, isMultiple, maxListenersCount) => {
 			Array.from({length: processesCount}, () => execa('empty.js', getComplexStdio(isMultiple))),
 		);
 		await setImmediate();
+		await setImmediate();
```

Comment:
Brittle test that depends on the number of ticks needed by Execa to spawn a child process. :-/